.storybook/preview.css

```
@@ -0,0 +1,41 @@
+  --color-surface-hover: #253456;
+  --color-border: #2a3a5c;
+  --color-text: #e0e0e0;
+  --color-text-secondary: #8892b0;
+  --color-text-muted: #5a6a8a;
+  --color-primary: #64ffda;
+  --color-primary-dim: #3d9e85;
+  --color-error: #ff6b6b;
+  --color-warning: #ffd93d;
+  --color-success: #6bff8e;
+  --font-mono: "JetBrains Mono", "Fira Code", "Cascadia Code", monospace;
+  --font-sans: "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)